### PR TITLE
refact(build): update go version to 1.14.7

### PR DESCRIPTION
**Pull request by mittachaitu**
Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

**What this PR does**:
CVE-2020-16845 has been reported for go versions earlier to 1.14.7,
this PR upgrades the go version for travis builds.

**Checklist**
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has breaking changes related information
- [ ] PR messages has upgrade related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] Tests updated

### CODE REVIEW

**Comment by kmova** [comment]
@mittachaitu - there is a fmt failure. Can you please check?

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ services:
 - docker
 language: go
 go:
-- 1.12.9
+- 1.14.7
```
Comment (prateekpandey14) [inline .travis.yml:15]: @mittachaitu we have to so some more refactring similar to zfs localpv, follow the Makefile changes from this PR https://github.com/openebs/zfs-localpv/pull/148/files

**Review by prateekpandey14 — COMMENTED** [review]
We have to refactor the Makefile bootstrap operation, any `go get` operation  inside the project root folder will modify the go.mod and other go-module files.

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**